nasa/harmony
# HARMONY-1433: As a harmony service provider, I want harmony to only allow associating collections with services using UMM-S/UMM-C associations so that there is only one place I need to look

## Jira Issue ID
HARMONY-1433

## Description
As a harmony service provider, I want harmony to only allow associating collections with services using UMM-S/UMM-C associations so that there is only one place I need to look

## Local Test Steps
1. Verify regular harmony request works
2. Change a service chain configuration in `services.yml` to have collections configured without `granule_limit` or `variables`, verify harmony prints out a proper error message similar to `Error: Collections cannot be configured for harmony service: harmony/service-example, use umm_s instead.` and harmony service does not start up.

## PR Acceptance Checklist
